src/scss/components/_fullscreen-pop-over.scss

```diff
@@ -58,7 +58,13 @@ $fullScreenPopOver-shadow-color      : rgba($black, 0.12);
     .c-fullScreenPopOver-header-actionSecond {
         top: 50%;
         position: absolute;
+        text-decoration: none;
```

Comment:
Hmm from an accessibility side of things is this ok? The underline visually makes it look like links IMO.

Reply:
Agree - this is something that Ash and Pieter-Jan Voskamp have covered before. 
I think Design / UX are going to do a a full review in the coming weeks but for now all is okay.